composer.json

```diff
@@ -34,9 +34,10 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^6.5",
-        "zendframework/zend-diactoros": "^1.6|^2.0",
```

Comment:
There is no good reason to use `zendframework/zend-diactoros`.

Best option is of course nyholm/psr7. But we already have guzzlehttp/psr7 installed.